circle.yml

```diff
@@ -13,5 +13,5 @@ test:
 
 general:
   artifacts:
-    - "build/sphinx/html"
+    - "doc/build/sphinx/html"
```

Comment:
Maybe slightly better to follow scikit-learn convention and use doc/_build/html. If you do that you need to update README.rst.

If you really want to follow the scikit-learn, most of the doc-related things are in the `doc/Makefile` folder (e.g. you can build the html doc from the root folder but not clean it. Full disclosure I am always in the `doc` folder when I work on the doc) but oh well your call ...

Reply:
I pushed an update that rename `doc/build` directory in `doc/_build` and add a Makefile in doc. One can now build the documentation from the root directory using `make doc` or from the `doc` directory by simply using `make` (since we are in the doc directory, no need to be specific).

The README has also been updated.